packages/core/package.json

```diff
@@ -1,4 +1,19 @@
 {
-  "name": "@zog/fxl.js",
-  "version": "0.0.1"
+  "name": "@01group/fxl",
+  "version": "0.0.3",
+	"description": "fxl.js is a data-oriented JavaScript spreadsheet library. It provides a way to build spreadsheets using modular, lego-like blocks.",
+	"homepage": "https://github.com/zero-one-group/fxl.js",
+  "dependencies": {
+    "tslib": "^2.0.0"
+  },
+  "keywords": [
```

Comment:
As a lib, it is always better to set your node engine so npm would warn you for using it with incompatible nodejs version. It is safe to assume that this package requires node 8+. Since it compiles to es5.
```
"engines": {
		"node": ">=8.0.0"
}
```